link/src/main/java/com/stripe/android/link/ui/signup/SignUpViewModel.kt

```diff
@@ -52,14 +53,14 @@ internal class SignUpViewModel @Inject constructor(
 
     val merchantName: String = args.merchantName
 
-    val emailController = SimpleTextFieldController.createEmailSectionController(prefilledEmail)
+    val emailController = EmailConfig.createController(prefilledEmail)
 
     val phoneController = PhoneNumberController.createPhoneNumberController(
         initialValue = prefilledPhone,
         initiallySelectedCountryCode = args.configuration.customerBillingCountryCode,
     )
 
-    val nameController = SimpleTextFieldController.createNameSectionController(prefilledName)
+    val nameController = NameConfig.createController(prefilledName)
```

Comment:
Could you add all the impacted `Config` classes that new could create a controller on their own in the PR description?

Reply:
Done!